build/build.sh

```diff
@@ -176,6 +190,20 @@ function GetLogCmd {
   echo $logCmd
 }
 
+function downloadMSBuildForMono {
+  if [[ -z $useSystemMSBuild && ! -e "$MONO_MSBUILD_DIR/MSBuild.dll" ]]
+  then
+    echo "** Downloading MSBUILD from $MSBUILD_DOWNLOAD_URL"
+    curl -sL -o "$MSBUILD_ZIP" "$MSBUILD_DOWNLOAD_URL"
+
+    unzip -q "$MSBUILD_ZIP" -d "$ArtifactsDir"
+    # rename just to make it obvious when reading logs!
+    mv $ArtifactsDir/msbuild $ArtifactsDir/mono-msbuild
```

Comment:
MONO_MSBUILD_DIR points to this path right?

Reply:
That's correct. I will update the line to use that.